src/test/regress/output/multi_copy.source

```diff
@@ -49,6 +49,7 @@ COPY customer_copy_hash (c_custkey, c_name) FROM STDIN
 WITH (FORMAT 'csv');
 ERROR:  duplicate key value violates unique constraint "customer_copy_hash_pkey_560048"
 DETAIL:  Key (c_custkey)=(2) already exists.
+CONTEXT:  COPY customer_copy_hash, line 4: ""
```

Comment:
We previously disabled these by resetting the error_context_stack before EndRemoteCopy, because the context doesn't provide any value at this point; it always points to the last line number.

(It's very hard to get the right line number for constraint violations, since the line number reported by the worker corresponds to the subset of rows that went into a given shard and the master doesn't know which is the offending line)